tests/instron_stress_rig.py

```diff
@@ -39,6 +40,15 @@ class Instron_stress_rigTests(unittest.TestCase):
     Tests for the Instron IOC.
     """
 
+    def _change_channel(self, name):
+        # Setpoint is zero-indexed
+        self.ca.set_pv_value("CHANNEL:SP", CHANNELS[name] - 1)
+        self.ca.assert_that_pv_is("CHANNEL.RVAL", CHANNELS[name])
+        self.ca.assert_pv_alarm_is("CHANNEL", self.ca.ALARM_NONE)
+        self.ca.assert_that_pv_is("CHANNEL:GUI", name)
+        self.ca.assert_pv_alarm_is("CHANNEL:GUI", self.ca.ALARM_NONE)
+
+
```

Comment:
This has been refactored a bit because the way the current control channel is handled in the IOC is now slightly different.